go.mod

```diff
@@ -42,7 +45,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.1
 	github.com/hashicorp/go-metrics v0.5.2
 	github.com/pokt-network/ring-go v0.1.0
-	github.com/pokt-network/shannon-sdk v0.0.0-20240605120544-ff4fc44e8d6c
+	github.com/pokt-network/shannon-sdk v0.0.0-20240606213254-42787a73b552
```

Comment:
Now that `shannon-sdk` is in a separate repo, I needed to bump the commit hash to the one that uses the new protobuf (from `poktroll` repo). Cross dependencies are hard. 
Opened a PR in `shannon-sdk` repo: https://github.com/pokt-network/shannon-sdk/pull/3/files